common-java-root/build.gradle

```diff
@@ -0,0 +1,43 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+project.ext.vstsUsername = System.getenv("ENV_VSTS_MVN_ANDROIDCOMMON_USERNAME") != null ? System.getenv("ENV_VSTS_MVN_ANDROIDCOMMON_USERNAME") : project.findProperty("vstsUsername")
+project.ext.vstsPassword = System.getenv("ENV_VSTS_MVN_ANDROIDCOMMON_ACCESSTOKEN") != null ? System.getenv("ENV_VSTS_MVN_ANDROIDCOMMON_ACCESSTOKEN") : project.findProperty("vstsMavenAccessToken")
+
+buildscript {
+    apply from: rootProject.file("../gradle/versions.gradle")
+}
+
+allprojects {
+    /*
+    Order of repositories is important.
+     */
+    repositories {
+        /*
+        This is the project main repository and contains all of the packages we publish.
+        This azure devops feed also has an upstream to Maven Central as well as several other
+        internal azure devops respositories
+
+        NOTE: This repository is not currently available to our public CI (Travis)
+         */
+        maven {
+            name "vsts-maven-adal-android"
+            url 'https://identitydivision.pkgs.visualstudio.com/IDDP/_packaging/Android/maven/v1'
+            credentials {
+                username project.vstsUsername
+                password project.vstsPassword
+            }
+        }
+        //Required for google published packages not published to maven central
+        google()
+        /*
+        Required for Android Studio / Android Tools (for example: linter) packages not published
+        to maven central.
+
+        NOTE: When building in Travis packages will be pulled from here
```

Comment:
This line is likely not required anymore assuming we're not building in Travis.

Reply:
Addressed here: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1468/commits/15543add15c080d0a4c4a1293ccd837aab1f3aa7